spaceR/spaceR/UIExtentions/UIErrorAlerts.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  UIErrorAlerts.swift
+//  spaceR
+//
+//  Created by Kieran Woodrow on 2022/03/11.
+//
+
+import Foundation
+
+import UIKit
+
+extension UIViewController {
+    
+    func displayErrorAlertForLogin(title: String, errorMessage: String, buttonTitle: String) {
+        let alert = UIAlertController(title: title, message: errorMessage, preferredStyle: UIAlertController.Style.alert)
```

Comment:
We are half way there in terms of making this generic can we maybe use an enum so that its not just an error alert and can be used for other use cases?